packages/esm-patient-orders-app/src/routes.json

```diff
@@ -97,5 +97,11 @@
       "component": "printLabResultModal",
       "name": "print-lab-results-modal"
     }
+  ],
+  "modals": [
+    {
+      "name": "edit-lab-results-modal",
+      "component": "editLabResultsModal"
+    }
   ]
```

Comment:
Consolidate this `modals` key with the one on line 95. We shouldn't have two separate `modals` keys.